nautobot_netbox_importer/tests/fixtures/3.7/summary.json

```diff
@@ -8199,9 +8813,26 @@
                 }
             }
         },
+        "extras.customfieldchoiceset": {
```

Comment:
Is this different from Nautobot's `extras.customfieldchoice` model?

Reply:
Yes, it's different.

NetBox allows to define single CustomFieldChoiceSet with multiple choices, and assign in to multiple CustomFields.

Nautobot allows to define multiple CustomFieldChoice and each assign to the single CustomfField.

Implemented here: https://github.com/nautobot/nautobot-app-netbox-importer/pull/131